src/test.ts

```diff
@@ -0,0 +1,16 @@
+const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJhdXRoZW50aWNhdGVkIiwiZXhwIjoxNjc3ODcxNTE0LCJzdWIiOiIwZmRmYzJiYi0wZTczLTQxNWYtOWIyMC00MGZjMzNhZjI1MmMiLCJlbWFpbCI6ImNkdWRhQGNoYW5uZWxhcGUuY29tIiwicGhvbmUiOiIiLCJhcHBfbWV0YWRhdGEiOnsicHJvdmlkZXIiOiJnb29nbGUiLCJwcm92aWRlcnMiOlsiZ29vZ2xlIl19LCJ1c2VyX21ldGFkYXRhIjp7ImF2YXRhcl91cmwiOiJodHRwczovL2xoMy5nb29nbGV1c2VyY29udGVudC5jb20vYS9BRWRGVHA3MjRjLTRXM0lmTnlpbVhROHFmcTl3U245T3I5cnJMVmVHMVdoej1zOTYtYyIsImVtYWlsIjoiY2R1ZGFAY2hhbm5lbGFwZS5jb20iLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwiZnVsbF9uYW1lIjoiQ2hyaXMgRHVkYSIsImlzcyI6Imh0dHBzOi8vd3d3Lmdvb2dsZWFwaXMuY29tL3VzZXJpbmZvL3YyL21lIiwibmFtZSI6IkNocmlzIER1ZGEiLCJwaWN0dXJlIjoiaHR0cHM6Ly9saDMuZ29vZ2xldXNlcmNvbnRlbnQuY29tL2EvQUVkRlRwNzI0Yy00VzNJZk55aW1YUThxZnE5d1NuOU9yOXJyTFZlRzFXaHo9czk2LWMiLCJwcm92aWRlcl9pZCI6IjExMjM1OTU4MjU5MzA3MTM4NDM5NCIsInN1YiI6IjExMjM1OTU4MjU5MzA3MTM4NDM5NCJ9LCJyb2xlIjoiYXV0aGVudGljYXRlZCIsImFhbCI6ImFhbDEiLCJhbXIiOlt7Im1ldGhvZCI6Im9hdXRoIiwidGltZXN0YW1wIjoxNjc3MDgwMjE1fV0sInNlc3Npb25faWQiOiI2NDI0NTM5ZC01NjM4LTQyZTUtYTM5NS1lMmRhZDllZTA1ZmMifQ._mdsFYmbqiWeHET8jxNJ8GWJc1G9LA-RGlF-rVXcnKs';
```

Comment:
If you name your file something along the lines xxxx.spec.ts it will run with the rest of the unit tests. Can you make this an actual unit test or just remove it altogether.

Reply:
@ryankazokas I never intended to include this file. This is a script I wrote to help me utilize supabase. I will be removing it when done.